expertise/create_dataset.py

```diff
@@ -85,17 +87,59 @@ def deduplicate_publications(self, publications):
                 deduplicated.append(pub)
 
         return deduplicated
+    
+    def get_pub_weight(self, pub=None, venueid=None, weight_specification=None):
+        if not venueid or not weight_specification:
+            return None
+            
+        # Get domain from either domain field or invitation prefix
+        domain = getattr(pub, 'domain', None)
+        if domain is None:
+            domain = pub.invitation.split('/-/')[0]  # API1 fallback to invitation
+        
+        # Return early on DBLP papers (venueid =/= domain) and non-accepted papers (domain not in venue_list)
+        if not (venueid == domain and domain in self.venue_list):
+            return None
```

Comment:
why do you return None here? it looks like this is the inOpenReview: false rule?

Reply:
I've updated the logic, it doesn't return None now and papers following `not (venueid == domain and domain in self.venue_list)` are now selected in the `inOpenReview: false` condition